docs/manual/about/under-the-hood/guarantees.md

```diff
@@ -20,15 +20,22 @@ support specific guarantees.
 
 ### At-Least-Once
 
-The `at-least-once` delivery guarantee ensures that an [event][docs.data-model]
-received by a Vector component will be delivered at least once. While rare, it
-is possible for an event to be delivered more than once. See the
-[Does Vector support exactly once delivery](#does-vector-support-exactly-once-delivery)
-FAQ below).
+The `at_least_once` delivery guarantee ensures that an
+[event][docs.data-model] received by a Vector component will be
+delivered at least once. For a source, this indicates that it will wait
+for all connected sinks to mark the event as delivered before
+acknowledging receipt of the event. For a sink, this indicates that it
+will attempt to retry the delivery until the events are either accepted
+or rejected and then signal the source with the results of that
+delivery.
+
+While rare, it is possible for an event to be delivered more than
+once. See the [Does Vector support exactly once
+delivery](#does-vector-support-exactly-once-delivery) FAQ below).
 
 <Alert variant="outlined" severity="warning">
 
-In order to achieve at least once delivery between restarts your source must
+In order to achieve at least once delivery between restarts your sink must
```

Comment:
Is this still true? If ack'ing is enable, and Vector dies while an event is in-flight, it won't be acknowledged, right?

Reply:
For those sinks that support it, yes, so this section needs to be adjusted to reflect that.